pkgdb/tests/search.bats

```diff
@@ -531,10 +531,10 @@ genParamsNixpkgsFlox() {
 @test "'pkgdb search' doesn't crash when run in parallel" {
   # We don't want other tests polluting parallel test runs so we do this test
   # with a unique cache directory.
-  run --separate-stderr sh -c 'PKGDB_CACHEDIR="$(mktemp -d)" parallel "sleep 0.{}; \"$PKGDB_BIN\" search --ga-registry --match-name hello" ::: $(seq 10)'
+  run --separate-stderr sh -c 'PKGDB_CACHEDIR="$(mktemp -d)" parallel "sleep 1.{}; \"$PKGDB_BIN\" search --ga-registry --match-name hello" ::: $(seq 5)'
```

Comment:
question: why do you increase the sleep time by 1 sec here?
question: if the problem was that with 10 runners you have contention because $0.1 == 0.10$, but we do want a larger number of parallel processes, why choose 5 over 9?

Reply:
Bad rebase, this is from the previous PR